pkg/cmd/pr/shared/params.go

```diff
@@ -208,7 +209,7 @@ func ListURLWithQuery(listURL string, options FilterOptions) (string, error) {
 
 func SearchQueryBuild(options FilterOptions) string {
 	q := search.Query{
-		Keywords: strings.Split(options.Search, " "),
+		Keywords: splitKeywords(options.Search),
```

Comment:
It feels weird to split the user's complete ES search string into keywords just to have `formatKeywords` join them into the same (hopefully) ES search string again. We could potentially avoid doing the dance and just pass the whole ES string through unchanged

Reply:
Yes, this is a much smarter idea 🙇